py/tests/dict.py

```diff
@@ -12,4 +12,8 @@
 a = repr({"a":"b","c":5.5})
 assert a == "{'a': 'b', 'c': 5.5}" or a == "{'c': 5.5, 'a': 'b'}"
 
+doc="check __iter__"
+a = {"a":"b","c":5.5}
+assert "a" in a
```

Comment:
Strictly speaking this test will no longer work when we implement the `__contains__` method for dict.

Do can you change it to something like

```python
l =  list(iter(a))
assert "a" in l
assert "c" in l
assert len(l) == 2
```

Reply:
This should be fixed now